ansible/playbooks/tasks/cluster-bootstrap.yaml

```diff
@@ -219,51 +223,73 @@
     op_default_timeout: "{{ (crm_op_options_show.stdout | regex_search('timeout=([0-9]*)', '\\1'))[0] | default('false') }}"
 
 - name: Enable SBD [sbd]
-  ansible.builtin.command: crm configure primitive rsc_iscsi_sbd stonith:external/sbd
+  ansible.builtin.command:
+    cmd: crm configure primitive rsc_iscsi_sbd stonith:external/sbd
   when:
     - sbd_stonith | string | lower  == 'false'
     - use_sbd
     - is_primary
 
 - name: Set stonith-timeout [sdb]
-  ansible.builtin.command: crm configure property stonith-timeout=144
+  ansible.builtin.command:
+    cmd: crm configure property stonith-timeout=144
   when:
     - stonith_timeout != '144'
     - use_sbd | bool
     - is_primary
     - cloud_platform_is_aws
 
 - name: Set stonith timeout [native - aws]
-  ansible.builtin.command: crm configure property \$id="cib-bootstrap-options" stonith-timeout=600s
+  ansible.builtin.command:
+    cmd: |
+      crm configure property
+      \$id="cib-bootstrap-options"
+      stonith-timeout=600s
   when:
     - stonith_timeout != '600s'
     - is_primary
     - not (use_sbd | bool)
     - cloud_platform_is_aws
 
 - name: Set stonith timeout [native - gcp]
-  ansible.builtin.command: crm configure property \$id="cib-bootstrap-options" stonith-timeout=600s
+  ansible.builtin.command:
+    cmd: |
+      crm configure property
+      \$id="cib-bootstrap-options"
+      stonith-timeout=600s
   when:
     - stonith_timeout != '300s'
     - is_primary
     - not (use_sbd | bool)
     - cloud_platform_is_gcp
 
 - name: Enable stonith
-  ansible.builtin.command: crm configure property \$id="cib-bootstrap-options" stonith-enabled=true
+  ansible.builtin.command:
+    cmd: |
+      crm configure property
+      \$id="cib-bootstrap-options"
+      stonith-enabled=true
   when:
     - stonith_enabled | string | lower != 'true'
     - is_primary
 
 - name: Disable stonith action [aws]
-  ansible.builtin.command: crm configure property \$id="cib-bootstrap-options" stonith-action=off
+  ansible.builtin.command:
+    cmd: |
+      crm configure property
+      \$id="cib-bootstrap-options"
+      stonith-action=off
   when:
     - stonith_action != 'off'
     - is_primary
     - cloud_platform_is_aws
 
 - name: Set rsc_defaults resource stickiness [aws/gcp]
-  ansible.builtin.command: crm configure rsc_defaults \$id="rsc-options" resource-stickiness=1000
+  ansible.builtin.command:
+    cmd: |
+      crm configure rsc_defaults
+      \$id="rsc-options"
```

Comment:
So is it `\$id` or `$id`? The `$` sign should go into the configuration I believe. See example here: https://documentation.suse.com/sle-ha/12-SP5/html/SLE-HA-all/cha-ha-manual-config.html#sec-ha-config-crm-global